airflow/providers/fab/auth_manager/cli_commands/user_command.py

```diff
@@ -212,10 +212,10 @@ def users_import(args):
 
     users_created, users_updated = _import_users(users_list)
     if users_created:
-        print("Created the following users:\n\t{}".format("\n\t".join(users_created)))
+        print(f"Created the following users:\n\t{'\\n\\t'.join(users_created)}")
```

Comment:
Can you undo these changes @eakmanrq ? 1) its breaking the image build and 2) we try to keep PRs isolated if possible (this is a provider, while the rest is the helm chart).

Reply:
Yeah I actually didn't make those changes. I think someone else updated my branch. Either way I will reverse it.

Reply:
Oh I see running style did this. Must be a misconfiguration on my end. Will fix.

Reply:
Fixed: https://github.com/apache/airflow/pull/41374/commits/ac2af646d7866446e26498a7a01d0bf58e814c32

Thanks for catching that.